src/main/_docs/setup/setup-cli.md

```diff
@@ -149,6 +149,12 @@ Connects the current terminal where the command is started to the terminal of a
 The syntax is `ssh <workspace-name> [machine-name]`
 The ssh connection will work only if there is a workspace ssh key setup. A default ssh key is automatically generated when a workspace is created.
 
+### sync
+Synchronize `/assembly` folder with the workspace folder.
```

Comment:
assembly folder? Maybe `Synchronize a local folder with a workspace machine folder`?

Also can we add `Refer to [local ide sync]({{base}}{{site.links["ide-sync"]}}) for additional information.`